OWNERS

```diff
@@ -1,16 +1,72 @@
-approvers:
-- enxebre
-- csrwng
-- sjenning
-- davidvossel
-- muraee
-- bryan-cox
-- jparrill
-options: {}
-reviewers:
-- enxebre
-- csrwng
-- sjenning
-- hasueki
-- rtheis
+filters:
+  ".*":
+    approvers:
+    - core-approvers
+    reviewers:
+    - core-reviewers
+    emeritus-approvers:
+    - davidvossel
+  # KubeVirt specific OWNERS
```

Comment:
I think we will need another 2 sections:
- For Openstack provider where Emilien Macchi.
- For Agent provider Nick Carboni and Crystal.
- For IBM Hide could be the choice.

WDYT?

Reply:
Might check with Emilien about who else can be added to OpenStack. Matt Booth may be added for instance.

Reply:
@stephenfin @mdbooth at least.

Thanks